server/templates/registration.html

```diff
@@ -7,11 +7,105 @@ <h1>Registration</h1>
     <h3>User details</h3>
     {{ user_form.as_p }}
     <h3>Player details</h3>
-    {{ player_form.as_p }}
```

Comment:
I'd like to continue using `.as_p` instead of manually writing down all the fields. 

If you need to get the parent element, you could do something like: `document.querySelector('#id_is_vaccinated').parentNode`